docs/versioned-plugins/inputs/http_poller-v4.0.1.asciidoc

```diff
@@ -115,7 +115,6 @@ This plugin supports the following configuration options plus the <<{version}-pl
 | <<{version}-plugins-{type}s-{plugin}-retry_non_idempotent>> |{logstash-ref}/configuration-file-structure.html#boolean[boolean]|No
 | <<{version}-plugins-{type}s-{plugin}-schedule>> |{logstash-ref}/configuration-file-structure.html#hash[hash]|No
 | <<{version}-plugins-{type}s-{plugin}-socket_timeout>> |{logstash-ref}/configuration-file-structure.html#number[number]|No
-| <<{version}-plugins-{type}s-{plugin}-ssl_certificate_validation>> |{logstash-ref}/configuration-file-structure.html#boolean[boolean]|No
```

Comment:
There was no option called ssl_certificate_validation, so I deleted the entries to fix the doc build.